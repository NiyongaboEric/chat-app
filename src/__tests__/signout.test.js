import React from 'react';
import { shallow } from 'enzyme';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Signout from '../components/user/signout/signout';
import { Redirect } from 'react-router-dom'

Enzyme.configure({ adapter: new Adapter() });


describe('Signout', () => {
  const wrapper = shallow(<Signout />);
  it('Should signout match snapshoot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('Should look like', () => {
    const compareMarkup=<Redirect to="/signin"/>;
    expect(wrapper.contains(compareMarkup)).toEqual(true);
  });

  it('Should redirect rendered ', () => {
    expect(wrapper.find(Redirect)).toHaveLength(1);
  });
});
