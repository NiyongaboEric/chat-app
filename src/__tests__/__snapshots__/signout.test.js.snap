// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signout Should signout match snapshoot 1`] = `ShallowWrapper {}`;
